traffic_ops/testing/api/v14/cachegroupsdeliveryservices_test.go

```diff
@@ -19,11 +19,8 @@ import (
 	"testing"
 )
 
-func TestDeliveryServicesCachegroups(t *testing.T) {
-	WithObjs(t, []TCObj{CDNs, Types, Tenants, Parameters, Profiles, Statuses, Divisions, Regions, PhysLocations, CacheGroups, Servers, DeliveryServices}, func() {
-		CreateTestCachegroupsDeliveryServices(t)
-		DeleteTestCachegroupsDeliveryServices(t)
-	})
+func TestCacheGroupsDeliveryServices(t *testing.T) {
+	WithObjs(t, []TCObj{CDNs, Types, Tenants, Parameters, Profiles, Statuses, Divisions, Regions, PhysLocations, CacheGroups, Servers, DeliveryServices, CacheGroupsDeliveryServices}, func() {})
```

Comment:
You appear to have disabled these unrelated tests; why?

Reply:
no they are just included in CacheGroupsDeliveryService now